Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -2859,13 +2859,13 @@ public void moveSeltectedPagesUp() {
         }
         List<String> newSelectionList = new ArrayList<>();
         for (Integer pageIndex : selectedPages) {
-            DocStructInterface firstpage = allPages.get(pageIndex - 1);
-            DocStructInterface secondpage = allPages.get(pageIndex);
-            switchFileNames(firstpage, secondpage);
+            DocStructInterface firstPage = allPages.get(pageIndex - 1);
+            DocStructInterface secondPage = allPages.get(pageIndex);
+            switchFileNames(firstPage, secondPage);
             newSelectionList.add(String.valueOf(pageIndex - 1));
         }
 
-        allPagesSelection = newSelectionList.toArray(new String[newSelectionList.size()]);
+        allPagesSelection = newSelectionList.toArray(new String[0]);
```

Comment:
Why did you changes this? Amount of entries is well known.

Reply:
Creating array without size is more efficient since java 6.